### ISSUE TITLE
When updating to the new update there is no navigation or homepage

### ISSUE THREAD

**Issue by Cookie-Monster-Coder** [AchillesPunks/Bingie.Mod.Matrix.Skin#2]
<img width="960" alt="Homescreen" src="https://user-images.githubusercontent.com/53286741/113252524-943afa00-9289-11eb-9109-73f9cfbf7e1c.png">

**Comment by AchillesPunks** [comment]
I'm not sure what you mean by this. Going to have to be a little more clear.

**Comment by cookie-monster-bit** [comment]
> 
> 
> I'm not sure what you mean by this. Going to have to be a little more clear.

Yeah, sorry. I installed this a couple of days ago with all the required supporting add-ons. I noticed that this Mod was updated today so I reinstalled it (installed it on top of the old version and had the notification of successful update) and am currently stuck on the page shown. I can't seem to bring up the sidebar menu or move in any direction.

**Comment by AchillesPunks** [comment]
Um, try not to update over the old. Try hitting start or s on the keyboard.

**Comment by AchillesPunks** [comment]
We are about to have it install everything automatically and everything
should work. Just give us a little more time
